src/main/java/org/myrobotlab/service/Sweety.java

```diff
@@ -880,7 +880,7 @@ public void startService() {
     adaFruit16cRight = (Adafruit16CServoDriver) Runtime.start("I2cServoControlRight", "Adafruit16CServoDriver");
     chatBot = (ProgramAB) Runtime.start("chatBot", "ProgramAB");
     htmlFilter = (HtmlFilter) Runtime.start("htmlFilter", "HtmlFilter");
-    mouth = (MarySpeech) Runtime.start("mouth", "MarySpeech");
+    mouth = (AbstractSpeechSynthesis) Runtime.start("mouth", "MarySpeech");

```

Comment:
this should be the interface, not the abstract class...

Reply:
agreed

Reply:
fixed in next commit